pager/src/main/java/com/google/accompanist/pager/Pager.kt

```diff
@@ -97,51 +108,62 @@ private val Measurable.page: Int
 fun HorizontalPager(
     state: PagerState,
     modifier: Modifier = Modifier,
+    reverseLayout: Boolean = false,
     @IntRange(from = 1) offscreenLimit: Int = 1,
-    content: @Composable PagerScope.(page: Int) -> Unit
+    decayAnimationSpec: DecayAnimationSpec<Float> = defaultDecayAnimationSpec(),
+    snapAnimationSpec: AnimationSpec<Float> = spring(stiffness = SnapSpringStiffness),
+    content: @Composable PagerScope.(page: Int) -> Unit,
 ) {
     require(offscreenLimit >= 1) { "offscreenLimit is required to be >= 1" }
 
-    val reverseScroll = LocalLayoutDirection.current == LayoutDirection.Rtl
-
-    val density = LocalDensity.current
-    val decay = remember(density) { splineBasedDecay<Float>(density) }
+    val isRtl = LocalLayoutDirection.current == LayoutDirection.Rtl
+    val reverseDirection = if (isRtl) !reverseLayout else reverseLayout
```

Comment:
If you do `if (isRtl) reverseLayout else !reverseLayout` to "reverse the reverse", do you still need the shenanigans with minuses in the rest of the logic?